framework/src/bundle/BundlePrivate.cpp

```diff
@@ -591,7 +591,7 @@ std::exception_ptr BundlePrivate::Start0()
         libHandle = BundleUtils::GetExecutableHandle();
       } else {
         if (!lib.IsLoaded()) {
-          lib.Load();
+          lib.Load(coreCtx->frameworkProperties);
```

Comment:
Here, you could use the new member of `coreCtx` which holds the `int` options and use the existing `SharedLibrary::Load(int)` function.

Reply:
Done.